staging/src/k8s.io/apimachinery/pkg/util/errors/errors.go

```diff
@@ -28,9 +28,12 @@ type MessageCountMap map[string]int
 
 // Aggregate represents an object that contains multiple errors, but does not
 // necessarily have singular semantic meaning.
+// Checks for specific errors can be performed via errors.Is().
```

Comment:
nit: update comment to include `As`.  An explanation of why `Is` and `As` are implemented but `Unwrap` is not may also be helpful.

Reply:
Done